cluster/src/main/scala/org/apache/spark/memory/SnappyUnifiedMemoryManager.scala

```diff
@@ -385,6 +385,15 @@ private object SnappyUnifiedMemoryManager extends Logging{
     testCallbacks.map(l => l.onPositiveMemoryIncreaseDueToEviction(objectName, bytes))
   }
 
+  def getMemorySize(conf: SparkConf): Long = {
+    val cache = Misc.getGemFireCacheNoThrow
+    if (cache ne null) {
+      OffHeapStorage.parseOffHeapMemorySize(
+        Misc.getGemFireCache.getDistributedSystem.getConfig.getMemorySize)
+    } else { // for local mode testing
+      conf.getLong("snappydata.store.memory.size", 0L)
```

Comment:
When can this happen? Local mode also will boot GemFire. I think it should be taken out.